WordPress/Classes/ViewRelated/Domains/Domain registration/DomainListCard.swift

```diff
@@ -0,0 +1,259 @@
+import SwiftUI
+
+struct DomainListCard: View {
+    struct DomainInfo {
+        let domainName: String
+        let domainHeadline: String
+        let state: State
```

Comment:
@alpavanoglu The property and its type should be renamed to `status`.

Reply:
Also, following our discussion in the team hangout, the property will be loaded asynchronously. So we'll probably need to wrap that `Status` enum in another enum, for example:

```swift
enum AsyncValue<T> {
    case loading
    case value(T)
}

struct DomainInfo {
        ...
        let status: AsyncValue<Status>
        ...
}
```

But we can handle this in another PR.

Reply:
My understanding is that `@Binding` works mostly in a SwiftUI environment. However, the `DomainListCard` will probably be embedded in a `UITableViewCell`, so how's `@Binding` going to work exactly?

Reply:
I think we can tackle it with Combine or something. I'm pretty sure we wouldn't need an enum of that sort